dev-haskell/stack/metadata.xml

```diff
@@ -6,20 +6,7 @@
 		<name>Gentoo Haskell</name>
 	</maintainer>
 	<use>
+		<flag name="developer-mode">By default, should extra developer information be output?</flag>
```

Comment:
If the `developer-mode` flag is removed from `IUSE`, this line will also need to be deleted.